src/util.h

```diff
@@ -275,6 +275,15 @@ extern char *git__strsep(char **end, const char *sep);
 extern void git__strntolower(char *str, size_t len);
 extern void git__strtolower(char *str);
 
+#ifdef GIT_WIN32
+GIT_INLINE(int) git__tolower(int c)
+{
+	return (c >= 65 && c <= 90) ? (c + 32) : c;
```

Comment:
I agree, that's much smarter.  I also changed this to compute the diff based on `'a' - 'A'` which I'm not sure that I love.  See what you think.